content/learn/book/welcome/community/_index.md

```diff
@@ -5,9 +5,27 @@ template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: List places where the community congregates, and differentiate
+Bevy would not be what it is without the support from the community. The community can be found in many locations such as:
+- [Bevy's Github Page](https://github.com/bevyengine/bevy)
+- [Bevy's Twitter](https://twitter.com/bevyengine)
+- [Bevy's Discord](https://discord.gg/bevy)
 
-TODO: link to awesome-bevy and discuss crate ecosystem
+More can be found at this website's [community page](https://bevyengine.org/community).
 
-TODO: link to CONTRIBUTING.md and talk about how Bevy is maintained by Cart plus a broad community of volunteers.
-Link to Cart's donation page
+Our community is very active and welcoming.
+
+## Third-Party Crates
+
+Bevy has a lot of third-party crates associated with it, such as [bevy_retrograde](https://github.com/katharostech/bevy_retrograde), [bevy_spicy_networking](https://github.com/CabbitStudios/bevy_spicy_networking) and [heron](https://github.com/jcornaz/heron).
+
+These are often published to [crates.io](https://crates.io) and use Bevy's plugin system, meaning technically, at this point, you can make your own thrid-party crates (although, they may not be entirely useful).
+
+There is a whole GitHub repository dedicated to showing awesome bevy assets, aptly named [bevy-assets](https://github.com/bevyengine/bevy-assets). Be sure to check it out!
```

Comment:
When talking about bevy-assets, we need to be clear that it includes plugins / crates as well as classical "assets" (sounds, models, sprites). As someone without a background in Unity etc. this was confusing to me at first.